sympy/tensor/index_methods.py

```diff
@@ -12,6 +12,18 @@
 
 from __future__ import print_function, division
 
+ """
```

Comment:
```suggestion
"""
```
There's also an indentation issue here